man/man7/zfsprops.7

```diff
@@ -1184,6 +1184,117 @@ and the minimum is
 .Sy 100000 .
 This property may be changed with
 .Nm zfs Cm change-key .
+.It Sy limit_bw_read Ns = Ns Ar size Ns | Ns Sy none
+.It Sy limit_bw_write Ns = Ns Ar size Ns | Ns Sy none
+.It Sy limit_bw_total Ns = Ns Ar size Ns | Ns Sy none
+Limits the read, write, or combined bandwidth, respectively, that a dataset and
+its descendants can consume.
+Limits are applied to both file systems and ZFS volumes.
```

Comment:
I would mention snapshots here. I know that snapshots _are_ filesystems or volumes, but I see enough people not quite grok that I'd want to be explicit: Maybe just:

> Limits are applied to filesystems and volumes, and to their snapshots.